config/schedule.rb

```diff
@@ -7,10 +7,10 @@
 # Every minute after 5am until 3am, check for departure inaccuracies.
 # Raw cron asterisk order: minute, hour, day of month, month, day of week
 every '* 0-2,5-23 * * *' do
-  rake 'bojangles:go'
+  rake 'bojangles:run'
 end
 
-# Every day, cache departure data.
+# Every day, check to see if we need new GTFS files.
 every :day, at: '4am' do
```

Comment:
It takes five-ish hours, but only once every few months. So whenever there's a new GTFS export, bojangles will go down for about five hours. Up to @sherson really.

Perhaps it might be preferable to try doing the preparation at midnight, so that when new GTFS data is available, we lose bojangles for a few hours before it goes live rather than after?

Reply:
I'm not worried about losing 5 hours of monitoring every few months, but starting at midnight sounds like a good idea.

Reply:
Right, my thinking was more than when a new publish goes live is exactly when we *would* want monitoring.